specification/schemas/enums/ErrorInvalidNHSNumber.yaml

```diff
@@ -0,0 +1,12 @@
+type: string
```

Comment:
This file name seems wrong for this content, if you look at all the error files, theyre single enums. This is more like NHS Channel Statuses.

Reply:
This file name is just copying the pattern used for the file it was duplicated from that has almost identical content

Reply:
I found it 

`type: string
enum:
  - CM_MISSING_VALUE
  - CM_NULL_VALUE
  - CM_INVALID_VALUE
  - CM_INVALID_NHS_NUMBER
  - CM_DUPLICATE_VALUE
  - CM_TOO_FEW_ITEMS
  - CM_ODS_CODE_REQUIRED
  - CM_CANNOT_SET_ODS_CODE
example: CM_INVALID_NHS_NUMBER
title: Enum_Error_InvalidValue`


my bad 😄

Reply:
I'm confused by this as well, why make a copy of `ErrorInvalidValue.yaml` and rename it to `ErrorInvalidNHSNumber.yaml` given that only one of these errors is related to the NHS Number?

Reply:
Sorry for unresolving this, I swear I'm usually not picky during reviews!

I don't understand why those errors should be under "Invalid NHS Number"

Reply:
It's specifically down to the example field on the enum - in case the `ErrorInvalidValue.yaml` is used elsewhere I've duplicated it to have the example as NHS Number in this file, if we understood the documentation framework better there might be a neat way to do this, but that would probably be a pretty deep dive to find out.

Reply:
This would also raise the question of why is the `ErrorInvalideValue.yaml` file named as such when it contains all definitions for all errors and not just the InvalidValue error?

Reply:
Happy to resolve but unless I am mistaken the only difference with with `ErrorInvalidValue.yaml` and `ErrorInvalidNHSNumber.yaml` is their title, and it doesn't make sense to me to group all those errors as "Invalid NHS Number" because that's just no true.